src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -407,142 +407,162 @@ internal virtual ImmutableArray<NamedTypeSymbol> GetAdditionalTopLevelTypes()
             return ImmutableArray<NamedTypeSymbol>.Empty;
         }
 
-        private void GetExportedTypes(NamespaceOrTypeSymbol sym, ArrayBuilder<NamedTypeSymbol> builder)
+        private void GetExportedTypes(NamespaceOrTypeSymbol symbol, int parentIndex, ArrayBuilder<Cci.ExportedType> builder)
```

Comment:
Fixed.

---
In reply to: [67203948](https://github.com/dotnet/roslyn/pull/12004#discussion_r67203948) [](ancestors = 67203948)